linux_docker_resources/entry_point.sh

```diff
@@ -12,8 +12,6 @@ export ORIG_GID=$(echo $ORIGPASSWD | cut -f4 -d:)
 export UID=${UID:=$ORIG_UID}
 export GID=${GID:=$ORIG_GID}
 
-ARCH=`uname -i`
```

Comment:
Hm, you know, I'm not 100% certain.  I should probably just leave it alone for now.  I'll revert that part.